.circleci/config.yml

```diff
@@ -24,12 +79,48 @@ jobs:
           # don't limit this to the number of allocated cores, the job is
           # likely to get OOMed and killed.
           GOOPTS: "-p 2"
-
+  publish:
+    docker:
+      - image: circleci/buildpack-deps:18.04
+    steps:
+    - checkout
+    - setup_remote_docker:
+        version: 18.06.0-ce
+    - publish_image:
+        image_name: prombench
+        dockerfile_path: "prombench/cmd/prombench/Dockerfile"
+    - publish_image:
+        image_name: funcbench
+        dockerfile_path: "funcbench/Dockerfile"
+    - publish_image:
+        image_name: amgithubnotifier
+        dockerfile_path: "tools/amGithubNotifier/Dockerfile"
+    - publish_image:
+        image_name: comment-monitor
+        dockerfile_path: "tools/commentMonitor/Dockerfile"
+    - publish_image:
+        image_name: fake-webserver
+        dockerfile_path: "tools/fake-webserver/Dockerfile"
+    - publish_image:
+        image_name: scaler
+        dockerfile_path: "tools/scaler/Dockerfile"
+    - publish_image:
+        image_name: load-generator
+        dockerfile_path: "tools/load-generator/Dockerfile"
 workflows:
   version: 2
   prombench:
     jobs:
-    - test:
+    - build_and_test:
         filters:
-          tags:
+          branches:
             only: /.*/
+    - publish:
+        requires:
+          - build_and_test
+        context: testinfra
+        filters:
+          branches:
+            only:
+              - dockerbuild
```

Comment:
why do we need this?

Reply:
Well, I was testing my CircleCI on branch `dockerbuild` instead of `master`. Will remove it.

Reply:
Unfortunately, I send this PR from branch `dockerbuild` of my fork, that's why CircleCI check can be passed here. If remove this line, I need to send another new PR.